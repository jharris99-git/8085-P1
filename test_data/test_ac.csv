175.45.176.0,0,149.171.126.13,0,unas,INT,3E-06,200,0,254,0,0,0,-,266666656,0,2,0,0,0,0,0,100,0,0,0,0,0,1424223248,1424223248,0.003,0,0,0,0,0,2,,, ,5,5,2,9,2,2,5
175.45.176.1,1199,149.171.126.14,514,udp,INT,3E-06,224,0,254,0,0,0,-,298666656,0,2,0,0,0,0,0,112,0,0,0,0,0,1424246117,1424246117,0.003,0,0,0,0,0,2,,, ,3,3,1,2,1,1,3
175.45.176.3,1043,149.171.126.15,53,udp,INT,5E-06,114,0,254,0,0,0,dns,91200000,0,2,0,0,0,0,0,57,0,0,0,0,0,1424246033,1424246033,0.005,0,0,0,0,0,2,,, ,23,23,13,13,13,10,23
175.45.176.0,5086,149.171.126.11,111,udp,INT,3E-06,168,0,254,0,0,0,-,224000000,0,2,0,0,0,0,0,84,0,0,0,0,0,1424224930,1424224930,0.003,0,0,0,0,0,2,,, ,1,1,1,19,1,1,1
175.45.176.1,1043,149.171.126.18,53,udp,INT,8E-06,114,0,254,0,0,0,dns,57000000,0,2,0,0,0,0,0,57,0,0,0,0,0,1424238890,1424238890,0.008,0,0,0,0,0,2,,, ,37,37,37,37,37,19,37
175.45.176.1,0,149.171.126.12,0,unas,INT,8E-06,200,0,254,0,0,0,-,72727272,0,2,0,0,0,0,0,100,0,0,0,0,0,1421930733,1421930733,0.011,0,0,0,0,0,2,0,0,0,10,10,6,10,6,6,6