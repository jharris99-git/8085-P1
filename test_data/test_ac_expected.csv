DoS
Fuzzers
Generic
Reconnaissance
Generic
Backdoor